src/components/Navbar/Navbar.jsx

```diff
@@ -0,0 +1,15 @@
+import styled from "styled-components";
+
+export const Navbar = styled.div`
+  width: 96px;
+  height: 37px;
+  top: 16px;
+  left: 688px;
+  gap: 0px;
+  opacity: 0px;
+
+  font-family: var(--font-redressed);
+`;
+export default function Navbar() {
+  return <div>Levantini</div>;
```

Comment:
lets change this to header component. A navbar is a element that has a bunch of links inside of them. We are only having a logo there. If therre were links we would include a navbar inside a header.
So for symantics you shouldnt put this as a div. You should put it as a <header>
Also you should have anotheer dic for Levantini. It is our logo. Make a logo component out of it